### DISCUSSION TITLE
Automatically clean old (listened) episodes?

### DISCUSSION THREAD

**Discussion by gbakeman** [advplyr/audiobookshelf#783] [Q&A] [answered]
I'm using ABS for a centralized, self-hosted podcast manager and it's working wonderfully.

Maybe I'm just missing something obvious, but is there any feature that can automatically purge old episodes that have been listened too? It's starting to build up and I think I might need to go through. Would be a nice feature to have if not already available!

**Comment by advplyr** [comment, marked as answer]
There is not but there is a feature request for it open

**Reply by gbakeman** [reply]
I think I found it in issue #615, thanks!
